Tests/SonarScanner.MSBuild.Common.Test/SonarPropertiesTests.cs

```diff
@@ -18,18 +18,18 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-using System.Collections.Generic;
-using System.Linq;
-using FluentAssertions;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
-
 namespace SonarScanner.MSBuild.Common.Test;
 
 [TestClass]
 public class SonarPropertiesTests
 {
     /// <summary>
-    /// Strings that are used to indicate arguments that contain non sensitive data.
+    /// Strings that are used to indicate arguments that contain non-sensitive data.
```

Comment:
Add something like NO secret keys or passwords are allowed to be here whatsoever. Every sensitive parameter specified in the begin step MUST be specified again in the END step.